packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -100,6 +104,28 @@ describe('openapi-request-builder', () => {
       { fetchCsrfToken: true }
     );
   });
+  it('executes a request using the timeout', async () => {
+    const delayInResponse = 1000;
+    const slowDestintaion = { url: 'https://slow-server.com' };
```

Comment:
```suggestion
    const slowDestintaion = { url: 'https://slow-server.example.com' };
```
[req] Use `example.com` for test URLs